css/site.styl

```diff
@@ -345,6 +346,24 @@ img
 
 // HEADER
 
+#header 
+  text-align: center
+
+.menu-item
+  margin: 0 $margin-tiny
+  letter-spacing: .1em
+  
+.editorial-title
```

Comment:
oh damn its not in the boiler yet :( well whatever you feel like. if a repeated thing across this site they imo worth adding